### PR TITLE
EOS-22311: Unit Test case Improvements

### PR DESCRIPTION
**Pull request by selvakumaar5496**
Signed-off-by: Selvakumar <selva.k.nambi@seagate.com>

# Problem Statement
- Unit Test case Improvements

# Design
-  Adding setup and teardown classmethods

# Coding 
-  Coding conventions are followed and code is consistent [Y/N]: Y
-  Confirm All CODACY errors are resolved [Y/N]: Y (all possible)

# Testing 
- [x] Confirm that Test Cases are added (for both the cases, fix and feature) [Y/N]: 
- [x] Confirm Test Cases cover Happy Path, Non-Happy Path and Scalability [Y/N]: 
- [x] Confirm Testing was performed with installed RPM [Y/N]:  

# Review Checklist 
  Before posting the PR please ensure
- [x] PR is self reviewed
- [ ] Is there a change in filename/package/module or signature [Y/N]: N
- [ ] If yes for above point, Is a notification sent to all other cortx components [Y/N]: N
- [x] Jira is updated
- [x] Check if the description is clear and explained. 
- [x] Check Acceptance Criterion is defined. 
- [ ] All the tests performed should be mentioned before Resolving a JIRA. 
- [ ] Verification needs to be done before marked as Closed/Verified 

# Documentation
- [ ] Changes done to WIKI / Confluence page

### CODE REVIEW

File: py-utils/test/message_bus/test_message_bus.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python3
 
 # CORTX Python common library.
-# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# Copyright (c) 2021 Seagate Technology LLC and/or its Affiliates
```
Comment (sachinpunadikar) [inline py-utils/test/message_bus/test_message_bus.py:4]: revert this change or make copyright year as 2020, 2021. I would prefer not to change the year. For new addition, the year will be current year. For changes in a file, no need to update the year

File: py-utils/test/message_bus/test_message_bus.py [hunk 2]
```diff
@@ -18,49 +18,214 @@
 
 
 import unittest
-from cortx.utils.message_bus import MessageBus, MessageProducer, MessageConsumer
+from cortx.utils.message_bus.error import MessageBusError
+from cortx.utils.message_bus import MessageBus, MessageBusAdmin, \
+    MessageProducer, MessageConsumer
 
+# Declared global to track threaded consumers
```
Comment (sachinpunadikar) [inline py-utils/test/message_bus/test_message_bus.py:25]: Number of Consumer Threads

**Comment by sachinpunadikar** [comment]
Fill up the PR template. I added few comments, work on that

File: py-utils/test/message_bus/test_message_bus.py (hunk 2 above)
Comment (selvakumaar5496) [reply py-utils/test/message_bus/test_message_bus.py:25]: done

File: py-utils/test/message_bus/test_message_bus.py (hunk 1 above)
Comment (selvakumaar5496) [reply py-utils/test/message_bus/test_message_bus.py:4]: Done

**Review by sachinpunadikar — APPROVED** [review]
+1

File: py-utils/src/utils/errors.py [hunk 3]
```diff
@@ -23,6 +23,7 @@
 ERR_SERVICE_UNAVAILABLE = 0x1503
 ERR_SERVICE_NOT_INITIALIZED = 0x1504
 ERR_NOT_INITIALIZED = 0x1505
+ERR_INVALID_MESSAGE_TYPE = 0x1506
```
Comment (ujjwalpl) [inline py-utils/src/utils/errors.py:26]: Why dont you use EINVAL ?

**Review by ujjwalpl — CHANGES_REQUESTED** [review]

File: py-utils/src/utils/errors.py (hunk 3 above)
Comment (selvakumaar5496) [reply py-utils/src/utils/errors.py]: Done. Reverted back to EINVAL

**Review by ujjwalpl — APPROVED** [review]

**Review by ujjwalpl — APPROVED** [review]

**Pull request merged**